geode-assembly/src/integrationTest/resources/assembly_content.txt

```diff
@@ -709,10 +709,15 @@ javadoc/org/apache/geode/compression/package-tree.html
 javadoc/org/apache/geode/connectors/jdbc/JdbcAsyncWriter.html
 javadoc/org/apache/geode/connectors/jdbc/JdbcConnectorException.html
 javadoc/org/apache/geode/connectors/jdbc/JdbcLoader.html
+javadoc/org/apache/geode/connectors/jdbc/JdbcPooledDataSourceFactory.html
```

Comment:
Just wonder what is the usage of `assembly_content.txt`?

Reply:
I think it is for external javadocs. Anytime we add new external javadocs they need to be added to this file